analyzers/its/regression-test.ps1

```diff
@@ -510,7 +510,7 @@ try {
     Build-Project-DotnetTool "Net7" "Net7.sln"
     Build-Project-DotnetTool "NetCore31WithConfigurableRules" "NetCore31WithConfigurableRules.sln"
     Build-Project-DotnetTool "akka.net" "src\Akka.sln"
-    Build-Project-DotnetTool "Automapper" "Automapper.sln"
+    Build-Project-DotnetTool "AutoMapper" "AutoMapper.sln"
```

Comment:
Educational: I wonder how the old one was working in the first place, given that the solution name is "AutoMapper.sln" and not "Automapper.sln". Maybe it's case-insensitive?

Reply:
`dotnet build` doesn't care but GitHub does.